### ISSUE TITLE
ERROR:libav.h264:error while decoding MB 57 3, bytestream -28

### ISSUE THREAD

**Issue by surfzoid** [jdillenkofer/camera_proxy#7]
Hi
very often i see this : 
ERROR:libav.h264:error while decoding MB 57 3, bytestream -28

**Comment by surfzoid** [comment]
I forget an more accuracy thing, use debug, so 
python3 -d ./src/main.py --debug
INFO:baichuan_udp_layer:Received discovery packet answer from 192.168.xxxxx
INFO:camera:Sending legacy login packet
INFO:camera:Receiving nonce packet
INFO:camera:Received nonce: xxxxxxxxxxxxx
INFO:camera:Sending modern login packet
INFO:camera:Send start video cmd
INFO:camera:Battery Percentage: 100
DEBUG:camera:Sending ping
DEBUG:camera:Received unhandled message. modern_message_id: 4115027578
DEBUG:camera:I Frame found
DEBUG:camera:AAC Frame found (l_size 226, r_size 226)
DEBUG:camera:AAC Frame found (l_size 247, r_size 247)
DEBUG:camera:AAC Frame found (l_size 260, r_size 260)
DEBUG:camera:P Frame found
DEBUG:camera:Received unhandled message. modern_message_id: 234

**Comment by jdillenkofer** [comment]
Hi,
the h264 stream probably contains invalid data, because of the unhandled messages.
I don't see any of these messages when i test it with my camera.

**Comment by surfzoid** [comment]
Yes i was confusing with unhandel message, i can check more if the problem occure with the two cam i have, also localy or through vpn.
If you want i can follow your instructions for more debugg, like ptrace or strace Linux prog, or the multi plate-forme Microsoft ide code studio, or visual studio
